unpackaged/config/qa/objects/Account/fields/SLAExpirationDate__c.field-meta.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This field is part of standard scratch org metadata - do we intend to use/capture it?